Jellyfin.Server/Filters/SecurityRequirementsOperationFilter.cs

```diff
@@ -6,86 +6,72 @@
 using Microsoft.OpenApi.Models;
 using Swashbuckle.AspNetCore.SwaggerGen;
 
-namespace Jellyfin.Server.Filters
+namespace Jellyfin.Server.Filters;
+
+/// <summary>
+/// Security requirement operation filter.
+/// </summary>
+public class SecurityRequirementsOperationFilter : IOperationFilter
 {
-    /// <summary>
-    /// Security requirement operation filter.
-    /// </summary>
-    public class SecurityRequirementsOperationFilter : IOperationFilter
+    private const string DefaultAuthPolicy = "Default";
+    private static readonly Type _attributeType = typeof(AuthorizeAttribute);
+
+    /// <inheritdoc />
+    public void Apply(OpenApiOperation operation, OperationFilterContext context)
     {
-        /// <inheritdoc />
-        public void Apply(OpenApiOperation operation, OperationFilterContext context)
-        {
-            var requiredScopes = new List<string>();
+        var requiredScopes = new List<string>();
 
-            var requiresAuth = false;
-            // Add all method scopes.
-            foreach (var attribute in context.MethodInfo.GetCustomAttributes(true))
+        var requiresAuth = false;
+        // Add all method scopes.
+        foreach (var authorizeAttribute in context.MethodInfo.GetCustomAttributes(_attributeType, true).Cast<AuthorizeAttribute>())
```

Comment:
Some controllers, like the CollectionController, specify the attribute on the class instead of the method. This check fails for those.

Reply:
L40-53 add the controller level scopes

Reply:
Ok then something else is causing some issue. The `CreateCollection` method only contains `CollectionManagement`, not `DefaultAuthorization`.

Reply:
That makes sense, the endpoint only has that requirement since it is a subset of DefaultAuthorization

Reply:
How should I know which ones are subsets of which ones? The OpenAPI specification doesn't provide any data about that.
I think it will be better to include all parents.